src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -234,7 +234,7 @@ object Parsers {
         syntaxErrorOrIncomplete(expectedMsg(token))
       }
       if (in.token == token) in.nextToken()
-      in.offset
+      offset
```

Comment:
This change is dubious, but it seems to be what's in the mind of the original author. Otherwise, there's no need to define the variable `offset`.